application/client/src/app/ui/views/workspace/headers/component.ts

```diff
@@ -48,6 +52,7 @@ class RenderedHeader {
 export class ColumnsHeaders extends ChangesDetector implements AfterContentInit {
     public readonly Direction = Direction;
     public offset: number = 0;
+    public ngMore: string = 'more_horiz';
```

Comment:
no need to have it as a property, just put a value of it into `html` directly... that's not dynamic property